backend/model/message/src/main/java/co/airy/model/message/MessageRepository.java

```diff
@@ -15,4 +16,18 @@ public static Message updateDeliveryState(Message message, DeliveryState state)
     public static boolean isNewMessage(Message message) {
         return message.getUpdatedAt() == null;
     }
+
+    public static String getContent(Message message, Map<String, String> metadata) {
```

Comment:
I would be more explicit here. something like `resolveContent`? (just a suggestion, the main point is that `getContent` is not accurate and a little misleading)